packages/components/src/theme/tabs.scss

```diff
@@ -0,0 +1,33 @@
+.content-tabs {
+    .nav-tabs {
+        border-bottom: 0;
+        display: flex;
+
+        & > li {
+            flex: auto;
+
+            &:first-child {
+                max-width: 50%;
+            }
+
+            & > a {
+                background-color: inherit;
+                border: 3px solid transparent;
+                color: #999999;
```

Comment:
similar, do we have a scss variable for this color?

Reply:
Fixed.